applications/common/frontend/src/components/ContactForm/contactForm.js

```diff
@@ -18,25 +18,62 @@ const initForm = () => {
     document.addEventListener(window.KYANITE_ON_DOM_CONTENT_LOAD, () => {
         Array.from(document.getElementsByClassName("contact-form"))
         .forEach(function(contactForm) {
+
+            const clsFormSubmitted = 'form-submitted';
+            const clsFormSubmitSuccess = 'form-success';
+            const clsFormSubmitFailure = 'form-failure';
+
+            const addFormClass = (clsName) => {
+                !contactForm.classList.contains(clsName) && contactForm.classList.add(clsName);
+            }
+
+            const removeFormClass = (clsName) => {
+                contactForm.classList.contains(clsName) && contactForm.classList.remove(clsName);
+            }
+
+            const toggleFormClass = (clsName, isAdd) => {
+                isAdd ? addFormClass(clsName) : removeFormClass(clsName);
+            }
+
+            const resetFormResult = () => {
+                removeFormClass(clsFormSubmitSuccess);
+                removeFormClass(clsFormSubmitFailure);
+            }
+
+            const setSubmitted = (isSubmitted) => {
+                toggleFormClass(clsFormSubmitted, isSubmitted);
+                !isSubmitted && resetFormResult();
+            }
+
+            const setSuccess = (isSuccess) => {
+                resetFormResult();
+                toggleFormClass(isSuccess ? clsFormSubmitSuccess : clsFormSubmitFailure, true);
```

Comment:
Why not use addFormClass directly, given that the second parameter is always set to true?